shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -976,18 +965,15 @@ - (void)dealloc {
 
   [self invalidateKeyboardAnimationVSyncClient];
   [self invalidateTouchRateCorrectionVSyncClient];
```

Comment:
Could you TODO this as well? We could use the same issue. All four of these calls are bad and should feel bad.

Reply:
And put warning comments on each of the methods that say not to use self and reference the issue.

Reply:
Done... in a pile of places.